apps.json

```diff
@@ -457,6 +457,16 @@
         ],
         "url": "https://github.com/YunoHost-Apps/democracyos_ynh"
     },
+    "diagramsnet": {
+        "branch": "master",
+        "category": "office",
+        "revision": "HEAD",
+        "state": "working",
+        "subtags": [
+            "draw"
+        ],
+        "url": "https://github.com/YunoHost-Apps/diagramsnet_ynh"
```

Comment:
This url doesn't exist yet. You can either change it to: https://github.com/Gofannon/diagramsnet_ynh, or transfer the project to `YunoHost-Apps` (I'll give you full access to the repo)

Reply:
> transfer the project to `YunoHost-Apps` (I'll give you full access to the repo)

Yes please, I cannot do it right now when I try to transfet it:

> Cannot find new owner 'https://github.com/YunoHost-Apps'

Reply:
Hmm, IDK. Only the owner of the project can transfer it... Can you give me an admin access to the github repo please?

Reply:
I still can't access to the options. You can add a new remove to your repo:

```
git remote rename origin old-origin
```

```
git remote add origin git@github.com:YunoHost-Apps/diagramsnet_ynh.git
``` 
or
```
git remote add origin https://github.com/YunoHost-Apps/diagramsnet_ynh.git
```
for https instead of ssh

Then push :-)

Reply:
Done, thanks